packs/feats/deathless-servant.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "classfeature",
         "description": {
-            "value": "<p>The servants of apocalypse know that it is impossible to cheat death, but your will is sufficient to help you outlive even your strongest enemies. The first time each day you would be reduced to 0 Hit Points but not immediately killed, you avoid being knocked out and instead of taking the damage, you regain a number of Hit Points equal to twice your level. Your @UUID[Compendium.pf2e.conditionitems.Item.Wounded] condition increases by 1.</p>\n<p>In addition, whenever you have the wounded condition, you gain a status bonus to your Armor Class equal to your wounded condition value and a status bonus to the damage of your Strikes equal to twice your wounded condition value.</p>"
+            "value": "<p>The servants of apocalypse know that it is impossible to cheat death, but your will is sufficient to help you outlive even your strongest enemies. The first time each day you would be reduced to 0 Hit Points but not immediately killed, you avoid being knocked out and instead of taking the damage, you regain a number of Hit Points equal to @Damage[(2*@actor.level)[healing]{twice your level}. Your @UUID[Compendium.pf2e.conditionitems.Item.Wounded] condition increases by 1.</p>\n<p>In addition, whenever you have the wounded condition, you gain a status bonus to your Armor Class equal to your wounded condition value and a status bonus to the damage of your Strikes equal to twice your wounded condition value.</p>"
```

Comment:
```suggestion
            "value": "<p>The servants of apocalypse know that it is impossible to cheat death, but your will is sufficient to help you outlive even your strongest enemies. The first time each day you would be reduced to 0 Hit Points but not immediately killed, you avoid being knocked out and instead of taking the damage, you regain a number of Hit Points equal to @Damage[(2*@actor.level)[healing]]{twice your level}. Your @UUID[Compendium.pf2e.conditionitems.Item.Wounded] condition increases by 1.</p>\n<p>In addition, whenever you have the wounded condition, you gain a status bonus to your Armor Class equal to your wounded condition value and a status bonus to the damage of your Strikes equal to twice your wounded condition value.</p>"
```